scripts/push-broker.sh

```diff
@@ -80,6 +80,11 @@ if [[ -z "$GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS" ]]; then
 fi
 echo "    GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS: $(echo "$GSB_SERVICE_CSB_AWS_S3_BUCKET_PLANS" | jq @json)" >>$cfmf
```

Comment:
nitpick (no-blocking): line 81 and 86 are repeated

Reply:
this was actually an issue. I had copy-pasted and not changed the names to match Postgres so push broker was not doing the right thing. 

Fixed it now